update info on creating eu-workspaces (#4454)
segmentio/segment-docs: src/guides/regional-segment.md

> info ""
> Use [this form](https://segment.typeform.com/to/k5ADnN5e#user_id=xxxxx){:target="_blank"} if you need to transition from your existing US-based workspace to an EU workspace.

> To create a workspace with a different data processing region, you can reach out to the Segment Support team, and they will be able to assist with enabling the feature on request. Once the feature has been enabled for a user, the user will be able to self-serve and create new workspaces in different data processing region by following these steps:

Reviewer: ```suggestion
To create a workspace with a different data processing region, reach out to the Segment Support team, and they will assist you with enabling the feature. Once the feature has been enabled, you'll be able to self-serve and create a new workspace in a different data processing region by following these steps:
```
